libs/design/src/molecules/sidebar/sidebar/sidebar.component.scss

```diff
@@ -1,8 +1,37 @@
+@use '../helper/variables';
+
 :host {
 	display: block;
-}
+	width: 240px;
+	flex-shrink: 0;
+	height: var(--sidebar-height);
```

Comment:
I'm not sure if this is necessary. I removed all instances of height on the sidebar components, added a height in the usage css and not seeing anything wrong.

Reply:
Agreed. I removed the requirement.